lib/logging_config.ini

```diff
@@ -0,0 +1,21 @@
+[loggers]
```

Comment:
These logger settings seem like a good fit for config file (config.ini). Why do we have them separately again? Please see the discussion and the approach taken in the [MLCAT](https://github.com/DeveloperCAP/MLCAT/pull/157).